README.md

```diff
@@ -109,7 +111,7 @@ The following are Azure custom checks added to the existing NHC suite of tests:
 | check_gpu_ecc | GPU Mem Errors (ECC) |  20000000 | 20000000 | 20000000 | NA | NA |
 | check_gpu_clock_throttling | GPU Throttle codes assertion | not present | not present | not present | NA | NA |
 | check_nccl_allreduce | GPU NVLink bandwidth | 228 GB/s | 228 GB/s | 460 GB/s | NA | NA |
-| check_ib_bw_gdr | IB device (GDR) bandwidth | 175 GB/s | 175 GB/s | 380 GB/s | NA | NA |
+| check_ib_bw_gdr | IB device (GDR) bandwidth | 180 GB/s | 180 GB/s | 380 GB/s | NA | NA |
```

Comment:
Why did this get changed to 180?

Reply:
We weren't boosting clocks before and now we are. With the clocks boosted the results are consistently higher